packages/ui-core/static/styles/gauzy/_gauzy-table-filters.scss

```
@@ -0,0 +1,128 @@
+// ── Smart-table filter row: collapsed by default, compact when open ─────────
```

Stylelint (17.14.0)

[error] Unknown word Smart-table (CssSyntaxError)

(CssSyntaxError)